src/GrpcProtos/capture.proto

```diff
@@ -382,6 +382,13 @@ message CGroupMemoryUsage {
   int64 active_file_bytes = 12;
 }
 
+message MemoryEventWrapper {
```

Comment:
nit: maybe drop Wrapper, just MemoryEvent?

Reply:
Thanks for all the suggestions! Changed to use `MemoryUsageEvent`.